### ISSUE TITLE
Weeds3D - retake video button not working

### ISSUE THREAD

**Issue by zackgrzywacz** [precision-sustainable-ag/On-farm-Protocols#314]
User in Vermont had an issue where whenever they tried to retake a video (using the retake video button), the weeds3D app would start taking a new video but then say "this video already exists". The previously existing video would then not show up in the weeds3D app. This occurred in the on-farm protocol. I will test to see if this is persistent in other tabs in the weeds3D app.

**Comment by brianwdavis** [comment]
Any fixes for this yet?

**Comment by elhenriq** [comment]
@mikahpinegar @mikahpinegar1 does any of the recent updates fix this?

**Comment by mikahpinegar** [comment]
yes this is fixed in the new version. not sure where it's at with being released

**Comment by saseehav** [comment]
@zackgrzywacz do we have a release plan for the newest version of Weeds3D for all the teams?

**Comment by zackgrzywacz** [comment]
@saseehav kinda. I've told some teams to put their tablets online so the update can be pushed to them, but I've definitely missed a number of them. I can send reminder emails to each project this week and do a big app update push

**Comment by saseehav** [comment]
@saseehav will work with on-farm and ce2 to deploy the newest app

**Comment by saseehav** [comment]
Will work on next deployment of app version this week. Issue with naming 1.6.4 versus 1.7 for example
<html>
<body>


1.7-field | 1 device | 7 | 0 | 7.14 MB | Min sdk 27 | 05/24/2023 01:39 PM | 5/18/23 | Actions
-- | -- | -- | -- | -- | -- | -- | -- | --
1.6.4-field | 4 devices | 10 | 0 | 7.15 MB | Min sdk 27 | 06/13/2023 02:05 PM | 5/24/23 | Actions
 

<br class="Apple-interchange-newline">
</body>
</html>
